src/definitions/collections/table.less

```diff
@@ -834,6 +834,350 @@ each(@colors, {
   width: @sixteenWide;
 }
 
+/*----------------------
+    Width per Device
+-----------------------*/
+
+/* Mobile Sizing Combinations */
+@media only screen and (min-width: @mobileBreakpoint) and (max-width: @largestMobileScreen) {
+  .ui.table th[class*="one wide mobile"],
+  .ui.table td[class*="one wide mobile"] {
+    width: @oneWide !important;
```

Comment:
Same `@oneWide` or similar used for all devices, which means width are same across devices.
I think it is required to define `@computerOneWide`, `@tabletEeightWide` and so on to realize responsive width per devices.

Reply:
Based on what was used in grid.less, I'm not sure that's what is supposed to happen. The widths are percentages, and just give you the ability to have different column widths per device:

`two wide computer one wide mobile`

This would mean that on a "computer screen" the column would be two wide, but on a "mobile screen" the column would only be one wide.